.storybook/pages/CadStep1/CadStep1.module.css

```diff
@@ -0,0 +1,70 @@
+/*------------------------------------*\
+    # CAD Step 1
+\*------------------------------------*/
+
+/**
+ * Page Layout
+ */
+.cad-step-1__layout {
+  padding: var(--eds-size-2);
+  @media all and (min-width: $eds-bp-md) {
+    padding: var(--eds-size-10);
+  }
+}
+
+/**
+ * Page Layout Left Section
+ */
+.cad-step-1__left-section {
+  @media all and (min-width: $eds-bp-md) {
+    padding-right: var(--eds-size-10);
+  }
+}
+
+/**
+ * Page Layout Left Section Heading
+ */
+.left-section__heading {
+  margin-bottom: var(--eds-size-6);
+}
+
+/**
+ * Page Layout Left Section Texts
+ */
+.left-section__text {
+  font-weight: var(--eds-font-weight-light);
```

Comment:
Let's remove this and flag for conversation. If this needs to be represented as a typography preset, then we need to apply that

Reply:
Sounds good to me, this was mostly done to match the mocks directly, but consuming product team will probably change as needed if we don't represent as a typography preset.